python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_graph/_digraph_group_chat.py

```diff
@@ -487,13 +493,17 @@ def _reset_execution_state(self) -> None:
     async def _apply_termination_condition(
         self, delta: Sequence[BaseAgentEvent | BaseChatMessage], increment_turn_count: bool = False
     ) -> bool:
-        """Apply the termination condition and reset graph execution state when terminated."""
+        """Apply the termination condition without resetting execution state.
```

Comment:
This method is no longer needed as it is the same as the base class implementation.